test/styles/RRR.R

```diff
@@ -39,38 +48,110 @@ fun_call(argument1,
 fun_call(
     argument1,
     argument2
-    )
+)
 
 ## 3
 fun_call(parameter = (
     stuff
-    ),
-         argument)
+),
+argument)
 
 ## 4
 fun_call(parameter = fun_argument(
-             stuff
-    ),
-         argument)
+    argument1
```

Comment:
It is actually right. But do we want to push the inner call argument all the way to beginning of the previous line?